### ISSUE TITLE
Unable to send forms on android 8.0.0

### ISSUE THREAD

**Issue by Chromicle** [opendatakit/skunkworks-crow#197]
## Software and hardware versions
Android 8, Honor 8x 

### Problem description
Unable to send blank/filled forms on Android 8.0.0. The screen gets stuck on Waiting for the hotspot to connect
i.e QR code is not appearing to scan if we want to connect through wifi entering the password it is giving toast that you should scan a QR code to connect this network 

### Steps to reproduce the problem
Just select any blank or filled form and try to send on an android 9 device

### Expected behavior
It should work as expected(we can send the forms)

**Comment by sarrocks1** [comment]
@Chromicle  Can you please give a detailed description . I have successfully run on honor 7x with android v8.0 and I m not facing any errors. Screenshots would help a lot.
Thank You.

**Comment by iamvasuh** [comment]
@Chromicle is this issue still coming on your device?

**Comment by Chromicle** [comment]
Yeah Iam facing that but not my device(friend) and I checked on realme 2 also it is still coming
